src/java.desktop/share/classes/java/awt/Robot.java

```diff
@@ -218,10 +218,56 @@ private static void checkIsScreenDevice(GraphicsDevice device) {
      * @param y         Y position
      */
     public synchronized void mouseMove(int x, int y) {
-        peer.mouseMove(x, y);
+        int leastXDiff = Integer.MAX_VALUE;
+        int leastYDiff = Integer.MAX_VALUE;
+        int finX1 = x;
+        int finY1 = y;
+        int finX2 = x;
+        int finY2 = y;
+
+        GraphicsEnvironment ge = GraphicsEnvironment.getLocalGraphicsEnvironment();
+        GraphicsDevice[] gs = ge.getScreenDevices();
```

Comment:
```suggestion
        GraphicsDevice[] gds = ge.getScreenDevices();
```

I found that `gds` is a more common name in our code and testbase. At least for me, I immediately read `gds` as GraphicsDeviceS, array of `GraphicsDevice`.
Same applies for the test.

P.S. It's not a call to action, it's more of an invitation for discussion.